src/Styleguide/Components/SelectedCareerAchievements.tsx

```diff
@@ -0,0 +1,308 @@
+import {
+  BorderBox,
+  Box,
+  Flex,
+  Link,
+  Sans,
+  Separator,
+  Serif,
+} from "@artsy/palette"
+import { SelectedCareerAchievementsArtistPage_artist } from "__generated__/SelectedCareerAchievementsArtistPage_artist.graphql"
+import { BlueChip } from "Assets/Icons/BlueChip"
+import {
+  hasSections,
+  highestCategory,
+} from "Components/Artist/MarketInsights/MarketInsights"
+import { Modal } from "Components/Modal/Modal"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface SelectedCareerAchievementsProps {
+  artist: SelectedCareerAchievementsArtistPage_artist
+  border?: boolean
+  Container?: (props: { children: JSX.Element }) => JSX.Element
+}
+
+const CATEGORIES = {
+  "blue-chip": "Blue chip",
+  "top-established": "Established",
+  "top-emerging": "Emerging",
+}
+const CATEGORY_LABEL_MAP = {
+  "blue-chip": "Represented by internationally reputable galleries.",
+  "top-established": "Represented by industry leading galleries.",
+  "top-emerging": "Represented by up-and-coming galleries.",
+}
+
+export interface ArtistInsightProps {
+  type: string
+  label: string
+  value?: string
+  entities?: ReadonlyArray<string>
+}
+
+class ArtistInsight extends React.Component<ArtistInsightProps> {
```

Comment:
This would probably be better as its own `ArtistInsight.tsx` file

Reply:
Agreed! Moved that and the learn more modal into their own files.